doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc

```diff
@@ -43,26 +81,27 @@ NOTE: If the default or assigned port is already in use, a new port is automatic
 NOTE: If the default or assigned port is already in use, a new port is automatically selected. Ensure that the ports are available and check the log for more information.
 
 * `org.uberfire.nio.git.ssh.cert.dir`: Location of the `.security` directory where local certificates are stored. Default value: Working directory.
+* `org.uberfire.nio.git.ssh.idle.timeout`: Sets the ssh idle timeout.
 * `org.uberfire.nio.git.ssh.passphrase`: Pass phrase used to access the public key store of your operating system when cloning git repositories with SCP style URLs. Example: `git@github.com:user/repository.git`.
 * `org.uberfire.nio.git.ssh.algorithm`: Algorithm used by SSH. Default value: `RSA`.
+* `org.uberfire.nio.git.gc.limit`: Sets the GC limit.
 * `org.uberfire.nio.git.ssh.ciphers`: A comma-separated string of ciphers. The available ciphers are `aes128-ctr`, `aes192-ctr`, `aes256-ctr`, `arcfour128`, `arcfour256`, `aes192-cbc`, `aes256-cbc`. If the property is not used, all available ciphers are loaded.
 * `org.uberfire.nio.git.ssh.macs`: A comma-separated string of message authentication codes (MACs). The available MACs are `hmac-md5`, `hmac-md5-96`, `hmac-sha1`, `hmac-sha1-96`, `hmac-sha2-256`, `hmac-sha2-512`. If the property is not used, all available MACs are loaded.
 +
 NOTE: If you plan to use RSA or any algorithm other than DSA, make sure you set up your application server to use the Bouncy Castle JCE library.
 
-* `org.uberfire.metadata.index.dir`: Place where the Lucene `.index` directory is stored. Default value: Working directory.
-* `org.uberfire.ldap.regex.role_mapper`: Regex pattern used to map LDAP principal names to the application role name. Note that the variable role must be part of the pattern because it is substituted by the application role name when matching a principal value to a role name.
-* `org.uberfire.sys.repo.monitor.disabled`: Disables the configuration monitor. Do not disable unless you are sure. Default value: `false`.
-* `org.uberfire.secure.key`: Password used by password encryption. Default value: `org.uberfire.admin`.
-* `org.uberfire.secure.alg`: Crypto algorithm used by password encryption. Default value: `PBEWithMD5AndDES`.
-* `org.uberfire.domain`: Security-domain name used by uberfire. Default value: `ApplicationRealm`.
-* `org.guvnor.m2repo.dir`: Place where the Maven repository folder is stored. Default value: `<working-directory>/repositories/kie`.
-* `org.guvnor.project.gav.check.disabled`: Disables group ID, artifact ID, and version (GAV) checks. Default value: `false`.
-* `org.kie.build.disable-project-explorer`: Disables automatic build of a selected project in Project Explorer. Default value: `false`.
-* `org.kie.verification.disable-dtable-realtime-verification`: Disables the real-time validation and verification of decision tables. Default value: `false`.
+{KIE_SERVER} nodes and {CONTROLLER}::
+
+Use these properties to configure how to connect to the {KIE_SERVER} nodes from the {CONTROLLER}.
```

Comment:
Suggested rewrite:
Use the following properties to configure the connections with the {KIE_SERVER} nodes from the {CONTROLLER}:

Reply:
Done